src/main/java/appeng/core/lib/module/AEModule.java

```diff
@@ -26,7 +26,18 @@
 	String value();
 
 	/**
-	 * TODO 1.10.2-MODUSEP - Document.
+	 * Use this to make your addon dependent of something using elix-x fancy dependency syntax:
+	 * Each module dependency statement is seperated by a semicolon (;).
+	 * To declare your module side-only, just use "client" or "server" as one dependency statement
+	 * You can also declare dependency on a mod or AE2 module.
+	 * For that, write your modifiers, seperated by a dash (-), followed by a colon (:), and the thing you want to depend on
+	 * modifiers are: 	"server"/"client" if you only depend on it on either client or server
+	 * 					*"required" is a REQUIRED modifier. Just use it!*
+	 * 					"force" is used if you do not require that dependency, aka soft-depend.
```

Comment:
Quite the opposite. If you declare force, game will crash if other modifiers failed.